score-vmware/src/main/java/io/cloudslang/content/vmware/constants/ErrorMessages.java

```diff
@@ -0,0 +1,30 @@
+package io.cloudslang.content.vmware.constants;
+
+/**
+ * Created by Mihai Tusa.
+ * 10/20/2015.
+ */
+public class ErrorMessages {
```

Comment:
@gituser173  It's worth waiting a little bit to see how much we will develop in VMware integration. If the class will grow bigger fast then I will break it in several Enum's  (e.g.: one enum for each group of errors).